tools/stats/import_test_stats.py

```diff
@@ -73,7 +73,7 @@ def is_cached_file_valid() -> bool:
 def get_slow_tests(
     dirpath: str, filename: str = SLOW_TESTS_FILE
 ) -> Optional[Dict[str, float]]:
-    url = "https://ossci-metrics.s3.amazonaws.com/slow-tests.json"
+    url = "https://ossci-metrics.s3.amazonaws.com/slow-tests-by-job.json"
```

Comment:
Just a note that now that we have the slow tests grouped by job, may be it's a good idea to see if the 60s threshold still makes sense.